mobile-app/app/screens/AppNavigator/screens/Loans/components/SymbolIcon.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import { getNativeIcon } from '@components/icons/assets'
```

Comment:
placing this component at common place will be more beneficial to use it in other screens like in `app/components/icons`

Reply:
Correct me if i am wrong, currently there are no use cases for this outside of loan yet, I'll move it whenever there is one

Reply:
We can reuse it in Token details screen, Balance screen, dex screen too. so that we can have common logic to get `NativeIcon`

Reply:
Actually the main purpose of creating this component is to allow rendering the icon directly within the render function, since `getNativeIcon()` is not a functional component by itself, .e.g:
```
{
  symbols.map((symbol): JSX.Element | null => (          
    <SymbolIcon
      key={symbol}
      symbol={symbol}
    />
  ))        
}
```
But I think it make sense to move this to generic folder so in future it can be used by other screens, thanks for the suggestion